packages/amplify-graphql-migration-tests/package.json

```diff
@@ -37,18 +37,18 @@
     ]
   },
   "devDependencies": {
-    "@aws-amplify/graphql-auth-transformer": "^2.1.0-cdkv2.1",
+    "@aws-amplify/graphql-auth-transformer": "^2.1.0-cdkv2.2",
```

Comment:
did you mean to include this too? or you are bumping category API & this too

Reply:
yes. this is intended. I updated PR description. All packages from category api repo are part of this bump.